meshnets/utils/data_processing.py

```diff
@@ -28,18 +31,64 @@ def _compute_edge_attributes(data: Data, remove_pos: bool = True) -> Data:
     return data
 
 
+def edge_mesh_to_graph(node_coordinates: np.ndarray,
+                       node_features: np.ndarray,
+                       edges: np.ndarray,
+                       node_labels: Optional[np.ndarray] = None) -> Data:
+    """Produce an undirected graph object from edge mesh data and compute
+    its edge features.
+
+    Args shape:
+        node_coordinates: (nb_nodes, nb_space_dim)
+        node_features: (nb_nodes, nb_node_features)
+        edges: (nb_edges, 2)
+        node_labels: (nb_nodes, nb_node_labes)
+
+    The graph includes node features, edges indices, edge attributes
+    and optionally node labels."""
+
+    node_coordinates = torch.Tensor(node_coordinates)
+    node_features = torch.Tensor(node_features)
+
+    # Ensure a bidirectional graph with no duplicated edges
+    #pylint: disable = no-value-for-parameter
+    edge_index = to_undirected(torch.LongTensor(edges.T))
+
+    if node_labels is not None:
+        node_labels = torch.Tensor(node_labels)
+
+    graph = Data(x=node_features,
+                 edge_index=edge_index,
+                 y=node_labels,
```

Comment:
Question: What are the implications of saving `y=None`?

Reply:
None is the default value so it is identical to not providing a value for y.